src/sockets/tests/matchmakingQueue.test.ts

```diff
@@ -37,29 +37,22 @@ describe('MatchmakingQueue', () => {
     expect(matchmakingQueue.getNumInQueue()).toEqual(0);
   });
 
-  it('Matches 7 people in queue with join window', () => {
+  it('Matches 7 people in queue', () => {
```

Comment:
Can you leave this test as is but just skip it? 
We want this code to still exist and to still compile, but it's just currently disabled.

May be useful in the future.